pico/api/src/main/java/module-info.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2022 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Pico API module.
```

Comment:
This should explain what Pico is and where to start

Reply:
The readme will explain that - which will be in a future pr.

Reply:
`module-info.java` and `package-info.java` are the only things to be seen when you access Javadoc. So it is good to add some basic documentation to them (at least where to start).

Reply:
pushed updates to package-info; please review again.